### PR TITLE
Menu du jour

### PR DESCRIPTION
**Pull request by Quentin-Macquart**
Hey Heloise, 
I just add the files for "Le Menu du jour"

### CODE REVIEW

File: src/pages/Food/index.jsx [hunk 1]
```diff
@@ -1,3 +1,121 @@
...
+            <Tab>Desserts</Tab>
+            <Tab>Boissons</Tab>
+          </TabList>
+
+          <TabPanel>
+            {' '}
+            <FoodSlider typeOfFood={plats} />
+          </TabPanel>
+          <TabPanel>
+            {' '}
```
Comment (HeloiseFerla) [inline src/pages/Food/index.jsx]: {' '}  ????? to remove

**Review by HeloiseFerla — APPROVED** [review]
Good job
I have just leave a comment

**Review by HeloiseFerla — CHANGES_REQUESTED** [review]
Good job.
Please check my comment to modify your code.

**Comment by Quentin-Macquart** [comment]
Hey Heloîse, 
I pulled the dev, add some updates and deleted code you asked for. 
i'm staying available for potential modifications. 
Have a great day coding!

**Review by HeloiseFerla — DISMISSED** [review]
Good job !

**Review by HeloiseFerla — DISMISSED** [review]

**Review by HeloiseFerla — DISMISSED** [review]

File: src/pages/Food/FoodModal/index.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+function FoodModal() {
+  const [renderModal, setRenderModal] = useState(true);
+
+  const hideTheModal = () => {
+    setRenderModal(!renderModal);
+  };
+
+  return (
+    <>
+      {renderModal ? (
```
Comment (Hacene010) [inline src/pages/Food/FoodModal/index.jsx]: you can replace this by renderModal && ... to gain some lines of code

File: src/pages/Food/index.jsx [hunk 3]
```diff
@@ -1,3 +1,134 @@
...
+
 export default function Food() {
-  return <p>Food</p>;
+  const [renderModal, setRenderModal] = useState(false);
+
+  const makeTheModalAppear = () => {
+    setRenderModal(!renderModal);
+  };
+
+  const plats = [
```
Comment (Hacene010) [inline src/pages/Food/index.jsx]: Use english for you variables names plz

File: src/pages/Food/index.jsx [hunk 4]
```diff
@@ -1,3 +1,134 @@
...
+      img: 'https://www.lsa-conso.fr/mediatheque/7/4/4/000189447_5.jpg',
+    },
+    {
+      name: 'Coca zero',
+      price: '2€50',
+      img: 'https://www.lsa-conso.fr/mediatheque/7/4/4/000189447_5.jpg',
+    },
+  ];
+
+  console.log(renderModal);
```
Comment (Hacene010) [inline src/pages/Food/index.jsx]: forgot a console.log !!!!

File: src/pages/Food/index.jsx [hunk 5]
```diff
@@ -1,3 +1,134 @@
...
+
+  return (
+    <SFood>
+      <h1 className="menuOfDay"> Le menu du jour</h1>
+      <section className="choiceMenu">
+        {renderModal ? <FoodModal /> : ''}
```
Comment (Hacene010) [inline src/pages/Food/index.jsx]: renderModal && .... ?

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some changes to make. See the comments

File: src/pages/Food/index.jsx (hunk 4 above)
Comment (Quentin-Macquart) [reply src/pages/Food/index.jsx]: Done

File: src/pages/Food/index.jsx (hunk 3 above)
Comment (Quentin-Macquart) [reply src/pages/Food/index.jsx]: Done

File: src/pages/Food/FoodModal/index.jsx (hunk 2 above)
Comment (Quentin-Macquart) [reply src/pages/Food/FoodModal/index.jsx]: Done

**Review by Hacene010 — APPROVED** [review]
that's better ;)

**Review by HeloiseFerla — APPROVED** [review]

**Pull request merged**